### PR TITLE
adds support for anonymous azure blobs

### PR DESCRIPTION
**Pull request by rakelkar**
replaces #172 for #168 
/assign ellis-bigelow

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/212)

### CODE REVIEW

File: python/kfserving/README.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ KFServing supports the following storage providers:
     * If `GOOGLE_APPLICATION_CREDENTIALS` is not provided, anonymous client will be used to download the artifacts.
 * S3 Compatible Object Storage with a prefix "s3://"
     * By default, it uses `S3_ENDPOINT`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` environment variables for user authentication.
+* Azure Blob Storage with the format: "https://{STORAGE_ACCOUNT_NAME}.blob.core.windows.net/{CONTAINER}/{PATH}"
```
Comment (ellis-bigelow) [inline python/kfserving/README.md]: nit: $STORAGE_ACCOUNT_NAME, $CONTAINER, $PATH

**Comment by rakelkar** [comment]
Resolved nit and integration tested with tensorrt... will the initializer container magically update?

**Comment by rakelkar** [comment]
fixed (now omits prefixes from dest path)

**Pull request merged**